src/main/java/mate/academy/until/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.until;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/academy";
```

Comment:
Ensure that the database URL (`DB_URL`) is correct and that the database is running on the specified host and port. Adjust the URL if necessary to match your database configuration.